test/Libraries/Microsoft.Extensions.AI.OpenAI.Tests/IntegrationTestHelpers.cs

```diff
@@ -17,8 +17,8 @@ internal static class IntegrationTestHelpers
     {
         var configuration = TestRunnerConfiguration.Instance;
 
-        string? apiKey = configuration["OpenAI:Key"];
-        string? mode = configuration["OpenAI:Mode"];
+        string? apiKey = Environment.GetEnvironmentVariable("AI:OpenAI:ApiKey");
+        string? mode = Environment.GetEnvironmentVariable("AI:OpenAI:Mode");
```

Comment:
Why this change?  Do we need to update docs to set these environment variables?
https://github.com/dotnet/extensions/blob/dc8efa608fc461ed031eda7a639b2fddb5a2dafe/test/Libraries/Microsoft.Extensions.AI.Integration.Tests/README.md?plain=1#L11-L12

I'm guessing this might have been a temporary local change you made for testing.

Reply:
I meant to revert this. My keys are in different env var s locally.